modules/codegen/src/main/scala/com/twilio/guardrail/generators/JavaModule.scala

```diff
@@ -43,12 +38,17 @@ object JavaModule extends AbstractModule[JavaLanguage] {
   def asyncHttpClient(implicit Cl: CollectionsLibTerms[JavaLanguage, Target] with CollectionsLibType): ClientTerms[JavaLanguage, Target] =
     new AsyncHttpClientClientGenerator.ClientTermInterp
 
-  def extract(modules: NonEmptyList[String]): Target[Framework[JavaLanguage, Target]] = {
-    implicit val collections = new JavaCollectionsGenerator.JavaCollectionsInterp with JavaStdLibCollections
+  def extract(modules: NonEmptyList[String]): Target[Framework[JavaLanguage, Target]] =
     (for {
-      (protocol, model, enum, array, poly) <- popModule("json", ("jackson", jackson))
-      client                               <- popModule("client", ("async-http-client", asyncHttpClient))
-      (server, framework)                  <- popModule("server", ("dropwizard", dropwizard), ("spring-mvc", spring))
+      collections <- popModule(
+        "collections",
+        Some(new JavaCollectionsGenerator.JavaCollectionsInterp with JavaStdLibCollections),
```

Comment:
The reason `popModule` hasn't included a "default" option up until now is to force people to choose in configuration, in order to keep the configurations from turning into a convoluted mess. `framework` exists for this, but if somebody opts in to `modules` they should be forced to specify all of them explicitly.

I'd be in favor of a `helpMessage: String` or `releaseNotes: String` though

Reply:
The issue here is that this will break existing users. That is, they will have a working setup, then will upgrade guardrail to a new version, and then suddenly it will fail, saying they haven't specified all the modules they need to.

If you're ok with that, I'm fine trying to add a help message that makes it easier to figure out what's missing.

Reply:
Fixed this up to remove the default after all, and will add a note to MIGRATING after I merge a different PR that will undoubtedly conflict.